### ISSUE TITLE
migrator not responding to "n0sc_smooth" message

### ISSUE THREAD

**Issue by einbahnstrasse** [CNMAT/CNMAT-Externs#124]
Hi all — I've got the preferred version of the CNMAT externals from the Max Package Manager (just re-downloaded current version last week to be sure), and the migrator object only seems to be able to change the number of oscillators with the "n0sc" message. "n0sc_smooth 40," for instance in the help file, does not work on my end. I'd love to be able to continue to use this message if possible. Thanks!

**Comment by equilet** [comment]
Yes, this is an issue we are aware of and actively working on.  Thanks!
